### DISCUSSION TITLE
Searchable indexers

### DISCUSSION THREAD

**Discussion by yasiryagi** [subquery/subql#1006] [Ideas]
It would be good to have the ability to search for an indexer by name or address
![16](https://user-images.githubusercontent.com/4862448/167710585-4febbbb1-a264-4cd7-8029-855e8f17310a.PNG)
![17](https://user-images.githubusercontent.com/4862448/167710602-d13b4bf5-8aab-4396-ad06-60f4351f1186.PNG)

**Comment by mzxyz** [comment]
This feature is under development, please expect this feature will be included in the next explorer updates

**Comment by heyacherry** [comment]
Hello @yasiryagi 
Searchable indexers feature has already live. Thank you!
